axum-login-libsql/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+edition = "2021"
+name = "axum-login-libsql"
+version = "0.1.0"
+
+[dependencies]
+async-trait = "0.1.72"
+axum-login = {path = "../axum-login/" }
+
+[dependencies.libsql-client]
+branch = "chore/dep-up"
+default-features = false
+features = ["reqwest", "reqwest_backend", "local_backend", "http"]
+git = "https://github.com/Suyashtnt/libsql-client-rs"
```

Comment:
this should likely wait until https://github.com/libsql/libsql-client-rs/pull/32 is merged, so we can just point at a published version